src/components/Layout/Page.tsx

```diff
@@ -35,30 +40,26 @@ function PageInset(props: { children: React.ReactNode }) {
   )
 }
 
-interface SectionProps {
-  children: React.ReactNode
+interface SectionProps extends BoxProps {
+  inheritBackground?: boolean
```

Comment:
Does it really make sense to introduce the new `inheritBackground ` prop here? As far as I can tell you could achieve the same effect by passing `backgroundColor="inherit"`.

Reply:
You're probably right that it makes more sense that way. 
I changed it so that `<DialogBody>` now has an optional `backgroundColor` instead of `inheritBackground` prop.